.github/release-please-config.json

```diff
@@ -13,5 +13,13 @@
       ]
     }
   },
+  "changelog-sections": [
+    { "type": "feat", "section": "Features" },
+    { "type": "feature", "section": "Features" },
```

Comment:
Might actually remove this since it's a bit redundant and it feels like providing an option that I don't actually want anyone to use :thinking: